source/manual/howto-use-restored-db-in-app.html.md

```diff
@@ -0,0 +1,83 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: How to use a restored database in an app
+section: Backups
+layout: manual_layout
+parent: "/manual.html"
+---
+
+Once you have [restored a database in AWS RDS](/manual/howto-backup-and-restore-in-aws-rds.html),
```

Comment:
We'll need to point to this new doc from the one above ^.
Or, there's an argument for keeping them both in the same doc - I can't think of a situation where you wouldn't want to point the app to the restored DB.

Reply:
Yep I think it makes sense to incorporate this as a section into /manual/howto-backup-and-restore-in-aws-rds.html, rather than it being a separate page

Reply:
Good point both! Will move this into the backup and restore doc.